requests/maniphest_createtask.go

```diff
@@ -0,0 +1,14 @@
+package requests
+
+// ManiphestCreateTaskRequest represents a request to maniphest.createtask.
+type ManiphestCreateTaskRequest struct {
+	Title        string                        `json:"title"`
+	Description  string                        `json:"description"`
+	OwnerPHID    string                        `json:"ownerPHID"`
+	ViewPolicy   string                        `json:"viewPolicy"`
+	EditPolicy   string                        `json:"editPolicy"`
+	CCPHIDs      []string                      `json:"ccPHIDs"`
+	Priority     int                           `json:"priority"`
+	ProjectPHIDs []string                      `json:"projectPHIDs"`
+	Request
```

Comment:
Currently missing the final "auxiliary - optional dict" parameter. I couldn't determine what it's used for and as such what type in Golang it should have. Works fine without it.